src/Admin/BlockAdmin.php

```diff
@@ -98,6 +98,10 @@ protected function configureRoutes(RouteCollectionInterface $collection): void
         $collection->add('composePreview', '{block_id}/compose_preview', [
             'block_id' => null,
         ]);
+
+        if (!$this->isChild()) {
+            $collection->remove('create');
```

Comment:
seems like a BlockAdmin can't be created unless is child of page or is a sharedBlock (another admin)